### DISCUSSION TITLE
请问今天登录不知道为什么会一直提示1040报错？

### DISCUSSION THREAD

**Discussion by wcabc007** [BurnHair/docs#149] [技术支持]
![image](https://github.com/BurnHair/docs/assets/135686510/183f3164-e052-40dd-a5d8-07626069ad81)
今天不知为什么一直报错。

**Comment by Azad-Aihub** [comment]
同样 错误：Error1040：Too many connections  看了一下运行状态好像超负荷了

**Comment by letschips** [comment]
+1

**Comment by wdsfto** [comment]
+1

**Comment by ZEERDEER** [comment]
https://status.burn.hair/

**Reply by wdmzjxzl** [reply]
+1

**Comment by uclinch** [comment]
+1

**Comment by NAFInsights** [comment]
+1, please kindly help with this.

**Comment by kangjian99** [comment]
是不是被攻击了？

**Reply by vicacheung** [reply]
据说是硬盘满了

**Comment by ehtesamul99** [comment]
+1, please kindly help with this. Will it be solved soon?

**Comment by BennyThink** [comment]
修好了！
